src/draw/vg_lite/lv_vg_lite_decoder.c

```diff
@@ -12,6 +12,8 @@
 #if LV_USE_DRAW_VG_LITE
 
 #include "lv_vg_lite_utils.h"
+#include <stdlib.h>
+#include <string.h>
```

Comment:
```suggestion
#include "../../misc/lv_types.h"
```

Reply:
I left the standard headers in the 'not core' .c files on purpose. I didn't know which of this files (drawing and external libraries) require the standard includes through their dependencies.

Reply:
I understand.
For `vg_lite`, it's normally the case we can use standard includes, so it's fine to leave it here.